### ISSUE TITLE
Remove deprecated twisted.application.app.initialLog

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5480]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)| @jesstess reported|
|-|-|
|Trac ID|trac#5480|
|Type|task|
|Created|2012-02-12 13:45:33Z|
|Branch|https://github.com/twisted/twisted/tree/remove-initiallog-5480|

`twisted.application.app.initialLog` was deprecated in r23738 (Twisted 8.2.0) and should be removed.

Attachments:

* [5480.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/f26/f2616f5d6774efbfe00ab4c50f1c145cb605295c/6a941009a639d5d791a404dca9bb68e4d355e7d3.patch) (1328 bytes) - added by acdn01 on 2012-02-26 19:49:33Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5480 5480
type__task task
reporter__jesstess jesstess
priority__lowest lowest
milestone__ 
branch__branches_remove_initiallog_5480 branches/remove-initiallog-5480
branch_author__habnabit habnabit
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1329054333000000 1329054333000000
changetime__1331535762000000 1331535762000000
version__None None
owner__ 
cc__jesstess cc__acdn01
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="acdn01's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|acdn01 commented|
|-|-|

Test suite passes with these changes.

**Comment by twisted-trac** [comment]
|[<img alt="habnabit's avatar" src="https://avatars.githubusercontent.com/u/21168?s=50" width="50" height="50">](https://github.com/habnabit)<a name="note_3"></a>|@habnabit commented|
|-|-|

(In [33681]) Branching to 'remove-initiallog-5480'

**Comment by twisted-trac** [comment]
|[<img alt="habnabit's avatar" src="https://avatars.githubusercontent.com/u/21168?s=50" width="50" height="50">](https://github.com/habnabit)<a name="note_4"></a>|@habnabit commented|
|-|-|

(In [33682]) Applying patch from ticket.

Refs [#5480](https://github.com/twisted/twisted/issues/5480).

**Comment by twisted-trac** [comment]
|[<img alt="habnabit's avatar" src="https://avatars.githubusercontent.com/u/21168?s=50" width="50" height="50">](https://github.com/habnabit)<a name="note_5"></a>|@habnabit set status to `closed`|
|-|-|

(In [33683]) Merge remove-initiallog-5480: Remove twisted.application.app.initialLog

Author: acdn01
Reviewer: habnabit
Fixes: [#5480](https://github.com/twisted/twisted/issues/5480)

twisted.application.app.initialLog was deprecated in Twisted 8.2.0 and is now
removed.
